test/connection/test_case_test.rb

```diff
@@ -34,7 +34,9 @@ def test_url_params
   end
 
   def test_plain_cookie
-    connect cookies: { user_id: "456" }
+    cookies[:user_id] = "456"
```

Comment:
Does it make sense to cover it with "version-aware" tests for deprecated features as we did in #45?